ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -1455,6 +1455,84 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 */6 * * 0-5
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: osde2e
+  labels:
+    ci.openshift.io/generator: prowgen
+  name: periodic-ci-openshift-osde2e-main-hypershift-stage-e2e-default
```

Comment:
@MrSantamaria Should we remove the existing periodic job [osde2e-hypershift-stage](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml#L12)? This way we do not have the same job running x2,

Reply:
There's two different purposes for these pipelines. 
The periodic job is being used to inform the total health of Hypershift. We have seen that job catch the breaking changes in the environment. 

The PR Check is mainly for QE to have something to test against as changes are made to the test while they live in the OSDe2e repo, so to not break the informing pipeline.

Reply:
Ah I misunderstood your comment. 
Good catch, I should remove the handcrafted job since it is being switched to prowgen.